lib/salus/scanners/yarn_audit.rb

```diff
@@ -307,6 +503,14 @@ def deep_copy_wo_paths(vulns)
       vuln_list
     end
 
+    def is_major_bump(current, updated)
+      current_v = current.tr("^", "").tr("~", "").split('.').map(&:to_i)
```

Comment:
would this break with something like `~1.2.3-beta`?

Reply:
Yup let me update that, thinking of getting rid of all characters except . and number since in this case I only care about the major digit.